src/hooks/useStore.js

```diff
@@ -0,0 +1,34 @@
+import { useSelector, useDispatch } from 'react-redux';
+import {
+  decrement,
+  increment,
+  incrementByAmount,
+  reset,
+} from '../redux/slices/counter';
+
+const useStore = () => {
+  const count = useSelector((state) => state.counter.value);
+
+  const dispatch = useDispatch();
+
+  const handleIncrement = () => {
+    dispatch(increment());
+  };
+  const handleDecrement = () => {
+    dispatch(decrement());
+  };
+  const handleReset = () => {
+    dispatch(reset());
+  };
+  const handleIncrementByValue = (value) => {
+    dispatch(incrementByAmount(value));
+  };
```

Comment:
no this is not what I told you 
in this file no dispatch each dispatch everything but it will work if it's called

Reply:
can you elaborate on this please